tests/Set/Marketplace/MarketplacePluginTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\SyliusRector\Tests\Set\Marketplace;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+final class MarketplacePluginTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(string $file): void
+    {
+        $this->doTestFile($file);
+    }
+
+    /**
+     * @return Iterator<string>
```

Comment:
```suggestion
     * @return \Iterator<string>
```

https://github.com/Sylius/SyliusRector/pull/34/files#diff-4008864cef25645ad2077180e8e6e542d8d7825a0c27c0ae46b3dfc4ddd9a47dR20